cypress/support/commands.js

```diff
@@ -2,11 +2,10 @@ Cypress.Commands.add('hslLogin', () => {
   const AUTH_URI = 'https://hslid-uat.cinfra.fi/openid/token';
   const AUTH_SCOPE = 'email https://oneportal.trivore.com/scope/groups.readonly';
 
-  const HSL_TESTING_HSLID_USERNAME = process.env.HSL_TESTING_HSLID_USERNAME || '';
```

Comment:
If these are not used anymore in deployment, these keys reference should be removed from env-files

Reply:
Removed